### ISSUE TITLE
(Feature request) Add non-intrusive internet connection lost message

### ISSUE THREAD

**Issue by Myzel394** [zadam/trilium#2366]
### Preflight Checklist

- [X] I have searched the [issue tracker](https://www.github.com/zadam/trilium/issues) for a feature request that matches the one I want to file, without success.

### Describe feature

When you have no internet connection and try to open notes, you will always get these notification boxes. They are not really understandable by people who are not versatile with computers.

I would suggest creating a non-intrusive banner at the top that shows that the internet is missing (see screenshot example from protonmail.com)

![image](https://user-images.githubusercontent.com/50424412/142756466-08cb6cbc-ba8d-4778-8044-bbeee79d5462.png)

**Comment by zadam** [comment]
Hi, do you mean on server version? Since on desktop for opening notes you don't need internet connection. Internet is needed for sync only and with that lack of internet is silent (only sync status icon changes). It would be useful if you could provide a screenshot and more context. Thx.

**Comment by Myzel394** [comment]
Yes I mean the the server version.

Here's what we could implement:

![image](https://user-images.githubusercontent.com/50424412/142756946-da040886-fa49-4189-a87f-a16cb1219258.png)

**Comment by Myzel394** [comment]
@zadam what do you think?

**Comment by zadam** [comment]
I guess why not?

(although it seems a bit ... superfluous? I mean with webapps running in a browser there is a general understanding that they don't work offline)

**Comment by Myzel394** [comment]
> although it seems a bit ... superfluous? I mean with webapps running in a browser there is a general understanding that they don't work offline

Yes of course :D, but at least we don't get spammed with this box notification which is exactly in the center of the page and is kind of annoying once you realize you don't have any internet

**Comment by zadam** [comment]
Ok, I'm not against this if you're willing to submit a PR.

**Comment by Myzel394** [comment]
Btw, does anyone know if there's a name for this kind of message?

**Comment by eliandoran** [comment]
We added a toast for when the websocket is disconnected. It should also cover this case.
